FAIL
Kernel panic in run: PANIC at ../../tests/threads/tests.c:60 in run_test(): no test named "priority-donate-one"
Call stack: 0x800421402d 0x80042178ac 0x8004206677 0x80042067e1 0x800420612b
Translation of call stack:
0x000000800421402d: debug_panic (lib/kernel/debug.c:32)
0x00000080042178ac: run_test (tests/threads/tests.c:61)
0x0000008004206677: run_task (threads/init.c:252)
0x00000080042067e1: run_actions (threads/init.c:297)
0x000000800420612b: main (threads/init.c:125)
